express/src/controllers/sign-in.ts

```diff
@@ -116,16 +116,21 @@ export const forgotPasswordForm = async function (req: Request, res: Response) {
 };
 
 export const checkEmailPasswordReset = async function (req: Request, res: Response) {
+    await forgotPasswordCognitoCall(req);
+    res.render("check-email-password-reset.njk");
+};
+
+export const resendForgotPassword = async function (req: Request, res: Response) {
+    await forgotPasswordCognitoCall(req);
+    res.redirect("/create-new-password");
+};
+
+export const forgotPasswordCognitoCall = async function (req: Request) {
```

Comment:
```suggestion
const forgotPassword = async function (req: Request) {
```

This doesn't need to be exported as it's only used within this file, it's an internal method.
Cognito is abstracted away by the S4 service so we shouldn't make references to it here, including the error message.

Reply:
Done.